### PR TITLE
feature/add-aics-cluster-check

### PR DESCRIPTION
**Pull request by evamaxfield**
Instead of running the pipeline in full to test the cluster, @domingusj had the idea of just writing a single deep test file that would run a series of tests to check the various SLURM nodes and their working (or not working) with Dask.

These are pretty basic and can probably be improved but it's a start.

Unfortunately, this PR will be staying open for a while because I currently can't start an `srun` session... check Slack #cluster as Calysta also reported this.

### CODE REVIEW

File: scripts/README.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Scripts
+
+## check_aics_cluster.py
+
+A script to run deep checks on the AICS SLURM cluster and it's interactions with Dask.
```
Comment (anl) [inline scripts/README.md]: Nit:
```suggestion
A script to run deep checks on the AICS SLURM cluster and its interactions with Dask.
```

**Comment by anl** [comment]
LGTM, but I don't know if I have enough context to give it the review you need.

File: scripts/test_aics_cluster.py [hunk 2]
```diff
@@ -0,0 +1,207 @@
...
+
+    print("All checks complete")
+
+
+########################################################################################
+# Actual tests
+
+
+@pytest.mark.parametrize("cores_per_worker", [1, 2, 4])
+@pytest.mark.parametrize("n_workers", [12, 24, 32, 64, 128])
```
Comment (toloudis) [inline scripts/test_aics_cluster.py]: 128 too small?  do you want to use this to find the breaking point?

**Review by toloudis — APPROVED** [review]
I don't quite understand what kind of results this is supposed to generate but the code looks ok!

File: scripts/test_aics_cluster.py (hunk 2 above)
Comment (evamaxfield) [reply scripts/test_aics_cluster.py]: I don't particularly want to find the breaking point but I did change these in a recent commit to be from 12 - 256 with less midpoints.

**Pull request merged**